source/playbooks/NIST80053/bin/nist80053.ts

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env node
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import {
+  SecurityControlsPlaybookPrimaryStack,
+  SecurityControlsPlaybookMemberStack,
+} from '../lib/security_controls_playbook-construct';
+import { App, Aspects, DefaultStackSynthesizer } from 'aws-cdk-lib';
+import { AwsSolutionsChecks } from 'cdk-nag';
+import 'source-map-support/register';
+import { IControl } from '../../../lib/sharrplaybook-construct';
+
+// SOLUTION_* - set by solution_env.sh
+const SOLUTION_ID = process.env['SOLUTION_ID'] || 'undefined';
+const SOLUTION_NAME = process.env['SOLUTION_NAME'] || 'undefined';
+// DIST_* - set by build-s3-dist.sh
+const DIST_VERSION = process.env['DIST_VERSION'] || '%%VERSION%%';
+const DIST_OUTPUT_BUCKET = process.env['DIST_OUTPUT_BUCKET'] || '%%BUCKET%%';
+const DIST_SOLUTION_NAME = process.env['DIST_SOLUTION_NAME'] || '%%SOLUTION%%';
+
+const standardShortName = 'NIST80053';
+const standardLongName = 'nist-800-53';
+const standardVersion = '5.0.0'; // DO NOT INCLUDE 'V'
+
+const app = new App();
+Aspects.of(app).add(new AwsSolutionsChecks());
+
+// Creates one rule per control Id. The Step Function determines what document to run based on
+// Security Standard and Control Id. See cis-member-stack
+const remediations: IControl[] = [
```

Comment:
I don't see CloudFormation.1, CloudTrail.6, CloudTrail.7, maybe more. Make sure this matches with the runbooks you want to add to this stack, or you won't get automatic remediation EventBridge rules for those controls in the Admin stack for this standard.